tests/utils/test_client.py

```diff
@@ -39,6 +39,12 @@ def test_invalid_credentials():
         utils.client.create(**CONFIG["my_other_instance"], enforce_healthy=True)
 
 
+@mock_api()
+def test_store_auth_cookie():
+    my_client = utils.client.create(**CONFIG["my_instance_name"], store_auth_cookie=True)
+    assert my_client.session.auth is None
```

Comment:
Using this option does cause `my_client.session.auth` to be `None` but it also sets a cookie, which is actually the more important consequence.  Can you change this test to check that `my_client.session.cookies` has an auth cookie set here? The actual cookie value doesn't matter, since it will be different every time.